### PR TITLE
Update docs for kill command

### PR DESCRIPTION
**Pull request by MattyTheHacker**

### CODE REVIEW

**Comment by CarrotManMatt** [comment]
Could you share the reasoning for removing that check? How did you know it was that one that was dodgy? What issues was it causing? Do you know why it didn't perform as expected? What was the values of the variables instead?

**Comment by MattyTheHacker** [comment]
was just fucking around with it putting logging in different places and realised it never actually passed that block cos that check wasn't passing, tried removing it and it works fine. 

iirc the comparison was a Message object to an Interaction object, so it was just permanently failing.

File: cogs/kill.py [hunk 1]
```diff
@@ -75,7 +75,6 @@ async def kill(self, ctx: TeXBotApplicationContext) -> None:
             "interaction",
             check=lambda interaction: (
                 interaction.type == discord.InteractionType.component
-                and interaction.message == confirmation_message
```
Comment (CarrotManMatt) [inline cogs/kill.py:78]: This check should be:

```python
and (
    interaction.message == confirmation_message
    if isinstance(confirmation_message, discord.Message)
    else interaction == confirmation_message
)
```

**Review by CarrotManMatt — CHANGES_REQUESTED** [review]
Ah yes, I see. That's because `confirmation_message` has type `discord.Message | discord.Interaction`. It is important for us to perform this check so that we do not incorrectly assume confirmation because of a button press in the same channel that is wholly unrelated.

File: cogs/kill.py (hunk 1 above)
Comment (MattyTheHacker) [reply cogs/kill.py:78]: nope that still doesn't work

Comment (CarrotManMatt) [reply cogs/kill.py:78]: Dodgy check is fixed in #202

**Comment by CarrotManMatt** [comment]
This is now dependent upon #202

**Review by CarrotManMatt — APPROVED** [review]

**Review by LMBishop — APPROVED** [review]

**Pull request merged**